notebooks/220-yolov5-accuracy-check-and-quantization/220-yolov5-accuracy-check-and-quantization.ipynb

```diff
@@ -46,11 +46,13 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Inference FP32 model (OpenVINO IR)
</pre><p>not all systems has GPU device installed, could you please make benchmarking on GPU optional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
I mean if "GPU' in core.available_devices, then run benchmark on GPU or something similar

Reply:
Sure, I will make the GPU optional for the benchmark by modifying the code as follows.

```
from openvino.runtime import Core

core = Core()
device = 'GPU' if 'GPU' in core.available_devices else 'CPU'
print(f"Benchmark runnin on Device: {device}")

# Inference FP32 model (OpenVINO IR)
!benchmark_app -m {fp32_path}.xml -d {device} -api async
```

Reply:
this variant does not allow to compare CPU vs GPU performance in the same notebook without code modification, I think it will be better to keep CPU benchmarking part and just put GPU part under condition:

```
print("Inference FP32 model (OpenVINO IR) on CPU")
!benchmark_app -m  {fp32_path}.xml -d CPU -api async
if if 'GPU' in core.available_devices:
    print("Inference FP32 model (OpenVINO IR) on GPU")
    !benchmark_app -m  {fp32_path}.xml -d GPU -api async
```

Also sometimes benchmarking can be very long, -t parameter allow reduce benchmarking time, you can consider to specify it

Reply:
Thank you for your suggestion. I will keep both CPU and GPU but just put GPU part under condition. Also I will add the -t parameter with Time 15 on benchmark_app.